### ISSUE TITLE
Broken Link

### ISSUE THREAD

**Issue by jalbertbowden** [GSA/github-federal-stats#71]
The Header logo link in the About pag heree:
http://gsa.github.io/github-federal-stats/datasheets/about.html
Points to:
http://gsa.github.io/github-federal-stats/datasheets/index.html
Which throws 404; Happy to fork, fix, and pr...just not sure where to point it to 100%.
Pretty sure it should go here though:
http://gsa.github.io/github-federal-stats/index.html

**Comment by gbinal** [comment]
Yep - I think you're exactly right.  If you're game to PR, that'd be great.
